FixedConstraint[] fixed_constraints
CircleCircleConstraint[] circle_circle_constraints
CircleConeConstraint[] circle_cone_constraints
CircleCylinderConstraint[] circle_cylinder_constraints
CircleLineConstraint[] circle_line_constraints
CirclePlaneConstraint[] circle_plane_constraints
CirclePointConstraint[] circle_point_constraints
ConeCircleConstraint[] cone_circle_constraints
ConeConeConstraint[] cone_cone_constraints
ConeCylinderConstraint[] cone_cylinder_constraints
ConeLineConstraint[] cone_line_constraints
ConePlaneConstraint[] cone_plane_constraints
ConePointConstraint[] cone_point_constraints
CylinderCircleConstraint[] cylinder_circle_constraints
CylinderConeConstraint[] cylinder_cone_constraints
CylinderCylinderConstraint[] cylinder_cylinder_constraints
CylinderLineConstraint[] cylinder_line_constraints
CylinderPlaneConstraint[] cylinder_plane_constraints
CylinderPointConstraint[] cylinder_point_constraints
LineCircleConstraint[] line_circle_constraints
LineConeConstraint[] line_cone_constraints
LineCylinderConstraint[] line_cylinder_constraints
LineLineConstraint[] line_line_constraints
LinePlaneConstraint[] line_plane_constraints
LinePointConstraint[] line_point_constraints
PlaneCircleConstraint[] plane_circle_constraints
PlaneConeConstraint[] plane_cone_constraints
PlaneCylinderConstraint[] plane_cylinder_constraints
PlaneLineConstraint[] plane_line_constraints
PlanePlaneConstraint[] plane_plane_constraints
PlanePointConstraint[] plane_point_constraints
PointCircleConstraint[] point_circle_constraints
PointConeConstraint[] point_cone_constraints
PointCylinderConstraint[] point_cylinder_constraints
PointLineConstraint[] point_line_constraints
PointPlaneConstraint[] point_plane_constraints
PointPointConstraint[] point_point_constraints
